src/general_manager/measurement/measurementField.py

```
@@ -1,168 +1,275 @@
+        """
+        if value is None:
+            return None
+        if isinstance(value, str):
+            value = Measurement.from_string(value)
+        if isinstance(value, Measurement):
+            try:
+                return Decimal(str(value.quantity.to(self.base_unit).magnitude))
+            except pint.errors.DimensionalityError:
+                raise ValidationError(
+                    {self.name: [f"Inkompatible Einheit zu '{self.base_unit}'."]}
+                )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)